geometry/geometry-engine-simplify/README.md

```diff
@@ -1,29 +1,31 @@
-# Geometry Engine Simplify
+# Geometry engine simplify
```

Comment:
With the update of this README, we should also make sure the metadata files are up to date too (I don't see a change for those through in this PR?)

Reply:
Thanks for pointing that out, I've still not gotten used to the metadata files!